src/components/Checkbox/Checkbox.spec.jsx

```diff
@@ -17,77 +17,81 @@ describe('Checkbox', () => {
 	});
 
```

Comment:
most of the changes in here were just rearranging things a bit. It was a little weird, like it had duplicate `describe('props'` blocks and stuff